config/jobs/kubernetes/kops/helpers.py

```diff
@@ -136,8 +136,6 @@ def latest_aws_image(owner, name):
 
 distro_images = {
     'amzn2': latest_aws_image('137112412989', 'amzn2-ami-kernel-5.10-hvm-*-x86_64-gp2'),
-    'centos7': latest_aws_image('125523088429', 'CentOS 7.*x86_64'),
-    'centos8': latest_aws_image('125523088429', 'CentOS 8.*x86_64'),
```

Comment:
Users should be removed too. Also, not sure where u1804 is still used.

Reply:
good call. 1804 is also still used in the grid for older kops versions.

Reply:
ok, but not sure how valuable it is.